app/models/order.py

```diff
@@ -82,7 +82,7 @@ class Status:
     exp_year = db.Column(db.Integer)
     last4 = db.Column(db.String)
     stripe_token = db.Column(db.String)
-    stripe_session_id = db.Column(db.String)
+    stripe_payment_intent_id = db.Column(db.String)
```

Comment:
we should use in the same column

Reply:
we do not need this data specifically for any other purpose other than confirming the order status. Once order status is ` completed` it is of no further use. That's why dropping won't cause any problem to past orders.

Reply:
@pc-beast Still payment data should be available for every order irrespective of time.